iquip/apps/builder.py

```diff
@@ -314,8 +314,8 @@ def __init__(self, name: str, argInfo: Dict[str, Any], parent: Optional[QWidget]
               unit: The unit of the number value.
               scale: The scale factor that is multiplied to the number value.
               global_step: The step between values changed by the up and down button.
-              global_min: The minimum value. (default=float("-inf"))
-              global_max: The maximum value. (default=float("inf"))
+              global_min: The minimum value. (default=float("0"))
+              global_max: The maximum value. (default=float("99.99"))
```

Comment:
Like [builder.py](https://github.com/snu-quiqcl/iquip/blob/develop/iquip/apps/builder.py#L166), `float()` is unnecessary.